tests/tensor_search/integ_tests/test_custom_vector_index_normalize_embeddings_true.py

```diff
@@ -0,0 +1,352 @@
+        )
+
+
+        test_structured_index_request = cls.structured_marqo_index_request(
+            model=Model(name='ViT-B/32'),
+            normalize_embeddings=True,
+            distance_metric=DistanceMetric.Angular,
+            fields=[
+                FieldRequest(
+                    name="my_custom_vector",
+                    type="custom_vector",
+                    features=[FieldFeature.LexicalSearch, FieldFeature.Filter]),
+                FieldRequest(
+                    name="text_field",
+                    type="text",
+                    features=[FieldFeature.LexicalSearch]),
+
+                # For score modifiers test
+                FieldRequest(
+                    name="multiply",
+                    type="float",
+                    features=[FieldFeature.ScoreModifier]),
+
+                # For searchable_attributes and filter tests
+                FieldRequest(
+                    name="my_custom_vector_2",
+                    type="custom_vector",
+                    features=[FieldFeature.Filter]),
+                FieldRequest(
+                    name="my_custom_vector_3",
+                    type="custom_vector",
+                    features=[FieldFeature.Filter]),
+
+                # For lexical + searchable_attributes test
+                FieldRequest(
+                    name="exact_field",
+                    type="text",
+                    features=[FieldFeature.LexicalSearch]),
+                FieldRequest(
+                    name="barely_field",
+                    type="text",
+                    features=[FieldFeature.LexicalSearch]),
+
+                # For multimodal mixed field tests
+                FieldRequest(
+                    name="multimodal_text",
+                    type="text"),
+                FieldRequest(
+                    name="multimodal_image",
+                    type="image_pointer"),
+                FieldRequest(
+                    name="my_multimodal",
+                    type="multimodal_combination",
+                    dependent_fields={"multimodal_text": 0.4, "multimodal_image": 0.6})
+            ],
+            tensor_fields=["my_custom_vector", "my_custom_vector_2", "my_custom_vector_3",
+                           "text_field", "my_multimodal"]
+        )
+
+
+        cls.indexes = cls.create_indexes([
+            test_unstructured_index_request,
+            test_structured_index_request,
+        ])
+
+        cls.unstructured_custom_index = cls.indexes[0]
+        cls.structured_custom_index = cls.indexes[1]
+
+    def setUp(self):
+        super().setUp()
+        self.mappings = {
+            "my_custom_vector": {
+                "type": "custom_vector"
+            }
+        }
+
+        # Using arbitrary values so they're easy to eyeball
+        self.random_vector_1 = [1. for _ in range(512)]
+        self.normalized_random_vector_1_nd_array = np.array(self.random_vector_1)
```

Comment:
This one isn't normalized yet. Maybe it should be `self.random_vector_1_nd_array`

Reply:
changed